jekyll/_config.yml

```diff
@@ -173,4 +173,6 @@ analytics:
     go_button_clicked: go-button-clicked
 rollbar_env: development
 
+optimizely: "GfYszBpMkefBSwiiEkH3b3"
```

Comment:
maybe a YML comment to say which account this is? 🤷

Reply:
good idea